www/source/partials/docs/_using-hab-encryption.html.md.erb

```diff
@@ -23,20 +23,14 @@ Supervisors running in a ring can be configured to encrypt all traffic between t
 3. Start the Supervisor with the `-r` or `--ring` parameter, specifying the name of the ring key to use.
 
     ```shell
-    $ hab start --ring yourringname yourorigin/yourapp
-    ```
-
-    or, if using a standalone Supervisor package:
-
-    ```shell
-    $ hab-sup start --ring yourringname yourorigin/yourapp
+    $ hab sup run --ring yourringname yourorigin/yourapp
```

Comment:
This (and the rest of the instances in this file) need to be fixed as described here. See https://www.habitat.sh/docs/using-habitat/